app/helpers/application_helper.rb

```diff
@@ -1,2 +1,25 @@
 module ApplicationHelper
+  module ActionView
+    module Helpers
+      class FormBuilder
+        def date_select(method, options = {}, html_options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.<br>Unused method argument - `html_options`. If it's necessary, use `_` or `_html_options` as an argument name to indicate that it won't be used.